src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -49,7 +49,13 @@
       <v-card-actions class="pt-0">
         <v-spacer></v-spacer>
         <div class="form__btns">
-          <v-btn text color="primary"
+          <v-btn v-if="isAdministrativeDissolutionOrPutBackOn" text color="primary"
+            id="dialog-save-button"
+            :loading="saving"
+            @click.native="save()"
+            class="save-btn"
+          >File</v-btn>
```

Comment:
What is this doing? It looks like this v-btn is the same as the one below it.

Reply:
fixed